dspace-api/src/test/java/org/dspace/app/bulkedit/MetadataExportSearchIT.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.bulkedit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.common.io.Files;
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.app.launcher.ScriptLauncher;
+import org.dspace.app.scripts.handler.impl.TestDSpaceRunnableHandler;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.builder.ItemBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.Item;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MetadataExportSearchIT extends AbstractIntegrationTestWithDatabase {
+
+    private String subject1 = "subject1";
+    private String subject2 = "subject2";
+    private int numberItemsSubject1 = 30;
+    private int numberItemsSubject2 = 2;
+    private Item[] itemsSubject1 = new Item[numberItemsSubject1];
+    private Item[] itemsSubject2 = new Item[numberItemsSubject2];
+    private String filename;
+    private Collection collection;
+    TestDSpaceRunnableHandler testDSpaceRunnableHandler = new TestDSpaceRunnableHandler();
+    private ConfigurationService configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+
+        context.turnOffAuthorisationSystem();
+        Community community = CommunityBuilder.createCommunity(context).build();
+        collection = CollectionBuilder.createCollection(context, community).build();
+        String dspaceDir = configurationService.getProperty("dspace.dir");
+        filename = String.format("%s%smetadataExportSearch.csv", dspaceDir, File.separator);
```

Comment:
@benbosman and @joost-atmire :  Here's your problem in tests. This `filename` is being set to `[dspace.dir]/metadataExportSearch.csv` which in the test environment will be: `/home/runner/work/DSpace/DSpace/dspace-api/target/testing/dspace/metadataExportSearch.csv`  (As when tests run `dspace.dir` is set to that `target/testing/dspace` folder)

In our CI environment that is throwing a file not found:
```
java.io.FileNotFoundException: /home/runner/work/DSpace/DSpace/dspace-api/target/testing/dspace/metadataExportSearch.csv (No such file or directory)
```

It looks like in this PR, instead you've added `metadataExportSearch.csv` to `dspace-apir/src/test/resources/org/dspace/app/bulkedit/metadataExportSearch.csv` which is obviously a different directory then what you specify here.

If you fix the directory path here, it's possible this will all work correctly.  I'm not sure how this is working in your CI, but I think it's obvious that the bug is in this directory path.